docs/03-10-orchestration.md

```diff
@@ -1,6 +1,8 @@
 # Orchestration
 
-Orchestration is a mechanism that allows you to upgrade Kyma Runtimes. To create an orchestration, [follow this tutorial](08-05-orchestrate-kyma-upgrade.md). After sending the request, the orchestration is processed by `KymaUpgradeManager`. It lists Shoots (Kyma Runtimes) in the Gardener cluster and narrows them to the IDs that you have specified in the request body. Then, `KymaUpgradeManager` performs the [upgrade steps](03-03-runtime-operations.md#upgrade) logic on the selected Runtimes.
+> **NOTE:** Once all Kyma components become modules, they will be upgraded independently of one another and of Kyma Environment Broker (KEB). This document will be deprecated as irrelevant.
+
+Orchestration is a mechanism that allows you to upgrade SAP BTP, Kyma runtimes. To create an orchestration, [follow this tutorial](08-05-orchestrate-kyma-upgrade.md). After sending the request, the orchestration is processed by `KymaUpgradeManager`. It lists Shoots (Kyma runtimes) in the Gardener cluster and narrows them to the IDs that you have specified in the request body. Then, `KymaUpgradeManager` performs the [upgrade steps](03-03-runtime-operations.md#upgrade) logic on the selected Kyma runtimes.
```

Comment:
There is no `upgrade` anchor in the second link. Did you mean `update`?

Reply:
Yes, you're right. It should be `#upgrade-kyma`.